packages/elements/src/tree/elements/tree-item.ts

```diff
@@ -220,7 +225,7 @@ export class TreeItem<T extends TreeDataItem = TreeDataItem> extends ControlElem
    * @param changedProperties changed properties
    * @returns {void}
    */
-  protected willUpdate (changedProperties: PropertyValues): void {
```

Comment:
?

Reply:
My bad. Ran a clean install on it and it was fetching an older version of lit's reactive-element which still have `willUpdate` as public. Fixed.